.github/workflows/e2e.yml

```diff
@@ -0,0 +1,80 @@
+name: Integration Testing
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: cs9 x86_64 e2e
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTS_DIR: exported-artifacts
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y install podman
+
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          # Use githash of a tested commit instead of merge commit
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: rpm-artifacts
+          path: ${{ env.ARTIFACTS_DIR}}
+
+      - name: build hirte tests containers
+        run: |
+          podman build -f tests/Containerfile -t hirte-image .
+
+      - name: run hirte containers
+        run: |
+          podman run -d --net podman --rm -p 2020:2020 --name hirte-mgr \
+          $(podman images *hirte-image --format {{.ID}})
+          podman run -d --net podman --rm --name hirte-node-bar \
+          $(podman images *hirte-image --format {{.ID}})
+          podman run -d --net podman --rm --name hirte-node-foo \
+          $(podman images *hirte-image --format {{.ID}})
+          PODMAN_NET=$(hostname -I | awk '{print $1}')
+          echo "PODMAN_NET=$PODMAN_NET" >> $GITHUB_ENV
+
+      - name: run hirte orchestrator
+        run: |
+          podman exec -it hirte-mgr bash -c "man hirte.conf | \
+               grep -A 5 \"\[hirte\]\" | sed -e 's/^ *//g' \
+               -e 's/journald/stderr/' \
+               -e 's/agent.*/foo,bar/' > /etc/hirte/hirte.conf"
+          podman exec -d  hirte-mgr \
+               bash -c "hirte > mgr.log 2>&1 &"
+
+      - name: run hirte agent-bar
+        run: |
+          podman exec -it hirte-node-bar bash -c "man hirte-agent.conf | \
+               grep -A 6 \"\[hirte-agent\]\" | sed -e 's/^ *//g' \
+               -e 's/journald/stderr/' -e \"s/127.0.0.1/${{ env.PODMAN_NET }}/\" \
+               -e 's/agent-.*/bar/' > /etc/hirte/agent.conf"
+          podman exec -d hirte-node-bar \
+               bash -c "hirte-agent > bar.log 2>&1 &"
+          podman exec -d hirte-node-bar \
+               bash -c "pkill --signal TERM hirte-agent"
+          podman exec -it hirte-node-bar bash -c \
+               "grep \"Agent 'bar' connection attempt 0: success\" bar.log"
```

Comment:
Can't we use journalctl instead of manual log file?

```
podman exec -it hirte-agent-bar \
              sh -c 'journalctl | grep "connection attempt 0: success" | grep bar'
```

Reply:
@mwperina 
For the journalctl, I assume once hirte/hirte-agent will be controlled by systemd, journald will be an option
In the case of e2e better to test what rpm install, dont you think?